3rdparty/p256-m/CMakeLists.txt

```diff
@@ -9,6 +9,19 @@ target_include_directories(p256m
          $<INSTALL_INTERFACE:include>
   PRIVATE ${MBEDTLS_DIR}/library/)
 
+# Pass-through MBEDTLS_CONFIG_FILE and MBEDTLS_USER_CONFIG_FILE
```

Comment:
Since P256-M is a PSA driver, does it need the PSA config file(s)?

Reply:
It may do, but we currently do not support passing `MBEDTLS_PSA_CRYPTO_CONFIG_FILE` through CMake in general.

This would be a useful thing to have, but I think it is out of scope for this PR as it would involve adding new functionality across the whole repo rather than just the 3rdparty modules.

Reply:
The PSA config files only define `PSA_WANT_xxx` macros. Neither Everest nor p256-m consume those.